build/ppas.sh

```
@@ -0,0 +1,365 @@
+#!/bin/sh
+DoExitAsm ()
+{ echo "An error occurred while assembling $1"; exit 1; }
+DoExitLink ()
+{ echo "An error occurred while linking $1"; exit 1; }
+echo Assembling orderedstringmap
+/Library/Developer/CommandLineTools/usr/bin/clang -x assembler -c -target arm64-apple-macosx11.0.0 -o build/OrderedStringMap.o  -x assembler build/OrderedStringMap.s
+if [ $? != 0 ]; then DoExitAsm orderedstringmap; fi
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)